distributions/nuxt-legacy/bin/nuxt-legacy.js

```diff
@@ -24,3 +24,6 @@ require('@babel/register')({
 
 const suffix = require('../package.json').name.includes('-edge') ? '-edge' : ''
 require('@nuxt/cli' + suffix).run()
+  .catch((error) => {
+    require('consola').fatal(error)
```

Comment:
The error may be thrown before registering reporter that handles `fatal`. I suggest explicitly calling `process.exit(2)` after fatal error.

Reply:
ye, indeed while it would be rare it could in theory happen